video_xblock/static/js/studio-edit-utils.js

```diff
@@ -26,3 +26,27 @@ function fillValues(fields) {
     }
     return {values: values, defaults: notSet};
 }
+
+
+/**
+ * Display message with results of a performed action (e.g. a transcript manual or automatic upload).
+ */
+function showStatus(message, type, successMessageElement, errorMessageElement) {
```

Comment:
Don't use 'magic strings' (`'success'`, `'error'`). Change it ti boolean status `is_success` or create static variable for these statuses